x/evm/types/denom_config.go

```diff
@@ -26,6 +28,27 @@ const (
 	EighteenDecimals Decimals = 18
 )
 
+// ChainsCoinInfo is a map of the chain id and its corresponding EvmCoinInfo
+// that allows initializing the app with different coin info based on the
+// chain id
+var ChainsCoinInfo map[string]EvmCoinInfo = map[string]EvmCoinInfo{
+	utils.MainnetChainID: {
+		Denom:        types.BaseDenom,
+		DisplayDenom: types.DisplayDenom,
+		Decimals:     EighteenDecimals,
+	},
+	utils.TestnetChainID: {
+		Denom:        types.BaseDenomTestnet,
+		DisplayDenom: types.DisplayDenomTestnet,
+		Decimals:     EighteenDecimals,
+	},
+	utils.SixDecChainID: {
+		Denom:        types.BaseDenom,
+		DisplayDenom: types.DisplayDenom,
+		Decimals:     SixDecimals,
+	},
+}
```

Comment:
Probably it is better to just define the type here since this code will be used by different chains, and create an instance for it in where we configure the Evmos concrete types

Reply:
Not sure I understand, which type should we define here?

Reply:
This variable will be imported along with the x/evm package by all partners but this is something only specific to Evmos. I would move this one in the types folder instead.

Reply:
and moving to the types there will be cyclic dependency